cpp/configuration_Zp.h

```diff
@@ -1,23 +1,46 @@
 #include <iostream>
+#include <vector>
 
 using namespace std;
 
-void histoDefinition(map<TString, int> &nbins, map<TString, float> &low, map<TString, float> &high, map<TString, TString> &title) {
+void histoDefinition(map<TString, int> &nbins, map<TString, float> &low, map<TString, float> &high, map<TString, vector<float>> &binsx, map<TString, TString> &title) {
 
+  float tbsize = 0.05;
+  float tbedge = 0.0;
   // MET Variables
-  nbins.insert({"pfmet_pt", 120});
-  low.insert({"pfmet_pt", 5});
-  high.insert({"pfmet_pt", 605});
+  //nbins.insert({"pfmet_pt", 120});
+  //low.insert({"pfmet_pt", 5});
```

Comment:
Let's just remove the old low and high binning for all plots? The new ones that you implemented looks fine even if we binning with constant bin size.

Reply:
Done, leaving only the line with `nbins` for fixed binning commented out, with a comment explaining what the purpose of that commented-out line is.